adapters/resetdigital/resetdigital.go

```diff
@@ -4,305 +4,237 @@ import (
-		BidID: requestData.ID,
-		ImpID: imp.ID,
+	if imp.TagID == "" {
+		reqCopy.Imp[0].TagID = resetDigitalExt.PlacementID
 	}
 
-	if bidType == openrtb_ext.BidTypeBanner && imp.Banner != nil {
-		var tempH, tempW int64
-		if imp.Banner.H != nil {
-			tempH = *imp.Banner.H
-		}
-		if imp.Banner.W != nil {
-			tempW = *imp.Banner.W
-		}
-		if tempH > 0 && tempW > 0 {
-			rdImp.MediaTypes.Banner.Sizes = append(rdImp.MediaTypes.Banner.Sizes, []int64{tempW, tempH})
-		}
+	if len(request.Cur) == 0 || (len(request.Cur) == 1 && request.Cur[0] == "") {
+		reqCopy.Cur = []string{currencyUSD}
+	} else {
+		reqCopy.Cur = request.Cur
 	}
-	if bidType == openrtb_ext.BidTypeVideo && imp.Video != nil {
-		var tempH, tempW int64
-		if imp.Video.H != nil {
-			tempH = *imp.Video.H
-		}
-		if imp.Video.W != nil {
-			tempW = *imp.Video.W
-		}
-		if tempH > 0 && tempW > 0 {
-			rdImp.MediaTypes.Video.Sizes = append(rdImp.MediaTypes.Video.Sizes, []int64{tempW, tempH})
-		}
-		if imp.Video.MIMEs != nil {
-			rdImp.MediaTypes.Video.Mimes = append(rdImp.MediaTypes.Video.Mimes, imp.Video.MIMEs...)
-		}
+
+	reqBody, err := json.Marshal(&reqCopy)
+	if err != nil {
+		return nil, []error{&errortypes.BadInput{
+			Message: fmt.Sprintf("Error marshalling OpenRTB request: %v", err),
+		}}
 	}
-	if bidType == openrtb_ext.BidTypeAudio && imp.Audio != nil && imp.Audio.MIMEs != nil {
-		rdImp.MediaTypes.Audio.Mimes = append(rdImp.MediaTypes.Audio.Mimes, imp.Audio.MIMEs...)
+
+	uri := a.endpoint
+	if resetDigitalExt.PlacementID != "" {
+		uri = fmt.Sprintf("%s?pid=%s", a.endpoint, resetDigitalExt.PlacementID)
 	}
 
-	var bidderExt adapters.ExtImpBidder
-	var resetDigitalExt openrtb_ext.ImpExtResetDigital
+	reqHeaders := baseHeaders.Clone()
 
-	if err := json.Unmarshal(imp.Ext, &bidderExt); err != nil {
-		return resetDigitalRequest{}, err
-	}
-	if err := json.Unmarshal(bidderExt.Bidder, &resetDigitalExt); err != nil {
-		return resetDigitalRequest{}, err
+	reqs := []*adapters.RequestData{
+		{
+			Method:  http.MethodPost,
+			Uri:     uri,
+			Body:    reqBody,
+			Headers: reqHeaders,
+			ImpIDs:  []string{imp.ID},
+		},
 	}
-	rdImp.ZoneID.PlacementID = resetDigitalExt.PlacementID
-
-	reqData.Imps = append(reqData.Imps, rdImp)
 
-	return reqData, nil
+	return reqs, errs
 }
 
 func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.RequestData, responseData *adapters.ResponseData) (*adapters.BidderResponse, []error) {
-	if adapters.IsResponseStatusCodeNoContent(responseData) {
+	if responseData.StatusCode == http.StatusNoContent {
 		return nil, nil
 	}
 
-	if err := adapters.CheckResponseStatusCodeForErrors(responseData); err != nil {
-		return nil, []error{err}
+	if responseData.StatusCode >= http.StatusBadRequest && responseData.StatusCode < http.StatusInternalServerError {
+		return nil, []error{&errortypes.BadInput{
+			Message: fmt.Sprintf("Unexpected status code: %d. Run with request.debug = 1 for more info", responseData.StatusCode),
```

Comment:
Did you mean `request.test` instead? If not, I belive the logic inside `MakeRequest` makes is so the request body Prebid Server sends to your bid server, won't have a `{"debug": 1,..}` field, correct?